W-14400269 recorder 6.5.0 rns (#301)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-recorder-6.5.0-release-notes.adoc

= RPA Recorder 6.5.0 Release Notes

RPA Recorder enables you to record a business process for automation.

include::partial$release-notes/rn-last-version-message.adoc[] 

== November 28, 2023

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+1]

> * The *Retake* option now displays as a button in a dropdown that includes options to delay the screen freeze for capturing image-based mouse clicks. This assists with preparing collapsible menus and hidden elements before the screen capture freeze. User and mouse actions are not recorded during the delay.

Reviewer: ```suggestion
* The *Retake* option now displays as a button in a dropdown. Using the *Retake* option, you can delay the screen freeze for capturing image-based mouse clicks. This assists with preparing collapsible menus and hidden elements before the screen capture freeze. User and mouse actions are not recorded during the delay.
```

* You can now set the initial screen freeze delay directly from the image-based mouse clicks toolbar.

> * When recording a Web Task, the selected area now shows a red border.

Reviewer: ```suggestion
* When recording a Web Task, the selected area now has a red border.
```

* RPA Recorder now always generates an `RPARecorder` subfolder during installation. 
* RPA Recorder now uses TLSv1.3 by default, and HTTP is now disabled for all API requests. 
* When recording in Notepad++, long window titles and name attributes are now truncated in the Documentation section of the RPA Recording Editor. 
* Updated the UI of the Settings and Log files panels, as well as the Manual Activity and Comment dialogs.

== Other Important Changes

> * The attribute selection panel can no longer be opened when recording image-based mouse clicks in a Web Task. Once image-based mouse clicks recording is finished, the panel can be opened.

Reviewer 2: note: The panel becomes disabled and the user will not be able to collapse or expand it.

// == Compatibility
include::partial$release-notes/rn-headings.adoc[tag=compatibility,leveloffset=+1]

[%header%autowidth.spread]
|===
|Application |Version
|RPA Builder | 6.4.0 and later
|===

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+1]

[%header,cols="2a,1a"]
|===
| Issue Resolution | ID
|Restarting image-based mouse clicks after cancelling the Scaling Warning dialog no longer leads to multiple toolbars. |W-13941631
|When selecting the capture area on a Web Task while capturing image-based mouse clicks, the `Selected area cannot be null` error no longer occurs. |W-13966122
|The `Screen capture is null` error no longer occurs when closing the Display Settings Warning dialog. |W-13980627
|When capturing image-based mouse clicks during Web Task recording, clicks on menu items in dropdowns are now recorded. |W-14043830
|When capturing image-based mouse clicks during Web Task recording, the browser window now freezes properly when selecting a dropdown menu. |W-14036847

> |When capturing image-based mouse clicks, pressing Alt+F4 in the capture image step no longer leads to an error. |W-14025390

Reviewer: Should Alt+4 be in ``? I am honestly not sure.
